src/codeu/chat/DerbyStore.java

```diff
@@ -137,7 +204,7 @@ public void updateLastMessage(Message m) throws SQLException {
 				// Add each created user to the store object
 				allUsers.insert(userid, user);
 			}
-			System.out.println("Successfully retrieved users");
+			LOG.info("Accessed users table.");
 		}
 		catch (Exception ex) {
 			ex.printStackTrace();
```

Comment:
Oh! My bad. Ignore this then. I must be running an old version of the code, which is why I'm seeing errors when I run the tests. I'll update my code whenever this gets merged.